README.md

```diff
@@ -1,5 +1,8 @@
 # pylhe: Python LHE interface
 
+<img src="https://user-images.githubusercontent.com/2318083/194557871-b863d51a-b9b0-4884-9331-1ce37a195588.png" alt="drawing" width="250"/>
```

Comment:
Ah okay, ignore my previous comment about this not rendering on PyPI as this is using the GitHub CDN and so would work. Though I agree with Eduardo that this would be good to have as a local file that could be used in a similar manner like https://github.com/scikit-hep/pyhf/blob/eac527e4601080df3c75db0276c208031f67e00c/README.rst?plain=1#L1-L4